plugins/woocommerce/includes/class-wc-webhook.php

```diff
@@ -280,7 +281,7 @@ private function is_valid_resource( $arg ) {
 			$status = get_post_status( absint( $arg ) );
 
 			// Ignore auto drafts for all resources.
-			if ( in_array( $status, array( 'auto-draft', 'new' ), true ) ) {
+			if ( in_array( $status, array( OrderStatus::AUTO_DRAFT, 'new' ), true ) ) {
```

Comment:
Here also, I think perhaps we are interested in products and coupons, rather than orders.